kubernetes/echo.yaml

```diff
@@ -53,7 +56,7 @@ spec:
           "--backend=127.0.0.1:8081",
           "--service=SERVICE_NAME",
           "--rollout_strategy=managed",
-          "--service_account_key=/etc/esp/creds/service-account-creds.json"
+          "--service_account_key=/etc/esp/creds/service-account-creds.json",
         ]
         ports:
           - containerPort: 8080
```

Comment:
NIT: be consistent to all include/not include `ports` in the section shown on doc?

Reply:
Yes, removed this file